pages/Documents/ConversationalAI/Third-PartyBots/MicrosoftDirectLine/introduction.md

```diff
@@ -0,0 +1,116 @@
+---
+pagename: Introduction
+redirect_from:
+  - bot-connectors-microsoft-bot-framework.html
+  - third-party-bots-microsoft-bot-framework.html
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: Microsoft Direct Line
+permalink: third-party-bots-microsoft-direct-line-introduction.html
+indicator:
+---
+
+### Overview
+The following documentation outlines the details of the Connector for Microsoft Direct Line.
+It describes how the connector is implemented and what limitations this imposes on bots that are added via this connector.
+It shows how activities need to be composed to successfully send messages and actions to the **Conversational Cloud**.
+
+It does not go into details on implementation details of such a bot.
+Microsoft provides different ways to implement chat bots like the 
+[Bot Framework SDK](https://github.com/microsoft/botframework-sdk#bot-framework-sdk-v4).
+A good starting point is the documentation on 
+[Azure Bot Services](https://docs.microsoft.com/en-us/azure/bot-service/).
+
+### Limitations
+
+{: .important}
```

Comment:
We can make them bullet points rather making each one separately  an important paragraph

Reply:
Use the important paragraph here on purpose to highlight the limitations. Is this a suggestion or a blocker?

Reply:
a suggestion as in other places of limitation we have thing bullet points